plugins/cloud-azure/src/test/java/org/elasticsearch/cloud/azure/AbstractAzureRepositoryServiceTestCase.java

```diff
@@ -38,22 +38,7 @@
 
 public abstract class AbstractAzureRepositoryServiceTestCase extends AbstractAzureTestCase {
```

Comment:
Well. Actually this test class is totally broken. Probably for months. May be we should better get rid of it entirely or fix that in another PR.